src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1546,23 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Address Mapping
+
```

Comment:
What an address mapping is? Is it dangerous to use it? (hint yes - it is not *typed*)

We need a short explanation about the underlying data-model.

I know such explanations are hard to write so please ask René or Duc to quickly help you on this.

Reply:
I have tried to write it. However, I'm still lack of confidence about this. Please correct me if I'm wrong

Reply:
I think you need to rewrite this. Please ask Duc or myself :)